src/test/ui/sanitize/address.rs

```diff
@@ -7,8 +7,6 @@
 // error-pattern: AddressSanitizer: stack-buffer-overflow
 // error-pattern: 'xs' (line 15) <== Memory access at offset
```

Comment:
Two lines were removed:

```suggestion
// error-pattern: 'xs' (line 13) <== Memory access at offset
```